docs/topics/extensions.rst

```diff
@@ -66,7 +66,7 @@ Disabling an extension
 
 In order to disable an extension that comes enabled by default (ie. those
 included in the :setting:`EXTENSIONS_BASE` setting) you must set its order to
-``None``. For example::
+``None``(or ``0``). For example::
```

Comment:
@jdemaeyer , I got tipped off track by [this](https://github.com/scrapy/scrapy/blob/master/scrapy/settings/__init__.py#L120). In the extensions [example](http://doc.scrapy.org/en/latest/topics/extensions.html#sample-extension), the `getbool` method is used to check if the extension is enabled.